packages/manager/modules/core/src/translate/ovh-ui-angular/index.js

```diff
@@ -121,6 +122,19 @@ angular.module(moduleName, [
           submitButtonLabel: $translate.instant('common_stepper_submit_button_label'),
           cancelButtonLabel: $translate.instant('common_stepper_cancel_button_label'),
         });
+
+        set(ouiPasswordConfiguration, 'translations', {
```

Comment:
Could be placed right after `ouiPaginationConfiguration`;)

Reply:
This is done :)